config/deploy/staging.rb

```diff
@@ -0,0 +1,23 @@
+############################################
+# Staging Server
+############################################
+
+set :stage, :staging
+set :stage_domain, "chappress-staging.chapman.edu"
+server "cprs-pre-wb01.chapman.edu", user: "wimops", roles: %w(web app db)
+set :deploy_to, "/usr/share/nginx/html"
```

Comment:
Where did this path come from? I usually expect the document root to be found under `/var/www`. Can we use `/var/www/html` as assumed [here](https://kb-smc.chapman.edu/?p=2086)?